telemeter-services/prometheus-telemeter.yaml

```diff
@@ -9,10 +9,10 @@ services:
     parameters:
       NAMESPACE: telemeter-production
       PROMETHEUS_IMAGE_TAG: v2.3.2
-      #PROMETHEUS_CPU_REQUEST: 100m
-      #PROMETHEUS_CPU_LIMIT: 1
-      #PROMETHEUS_MEMORY_REQUEST: 500Mi
-      #PROMETHEUS_MEMORY_LIMIT: 8Gi
+      PROMETHEUS_CPU_REQUEST: 100m
```

Comment:
we need to un-comment this block as this commit comes after the introduction of the resource requests.